research/model_description.ipynb

```diff
@@ -83,6 +75,39 @@
     "%matplotlib inline"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Contents of `npz` files:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "npz file contains 3 peices of data:\n",
+      "* 'time'\n",
+      "   shape - (273,), dtype - float64, example - array([1.4253...36489076e+03])\n",
+      "* 'derivative'\n",
+      "   shape - (273,), dtype - float64, example - array([  9.01...  0.        ])\n",
```

Comment:
could you explain this notation, please

Reply:
It is the corresponding array itself. You think it would be better to rephrase?

Reply:
yes, I think so. It hard to understand an example in this way.